commands/auth/2fa/generate_recovery_codes.js

```diff
@@ -0,0 +1,22 @@
+const cli = require('heroku-cli-util')
+const co = require('co')
+
+function * run (context, heroku) {
+  let password = yield cli.prompt('Password', {hide: true})
+  let headers = {'Heroku-Password': password}
+  let codes = yield heroku.post('/account/recovery-codes', {headers: headers})
+  cli.log('Recovery codes:')
+  for (var i in codes) cli.log(codes[i])
+}
+
+const cmd = {
+  description: 'generates and replaces recovery codes',
+  needsAuth: true,
+  run: cli.command(co.wrap(run))
+}
+
+module.exports = [
+  Object.assign({topic: 'auth', command: '2fa:generate-recovery-codes'}, cmd),
```

Comment:
@RasPhilCo this was my bad, it is [`auth:2fa:generate`](https://github.com/heroku/cli/blob/master/auth.go#L33) the rest should remain `generate-recovery-codes`